meetings/2022-08-18.md

```diff
@@ -0,0 +1,125 @@
+* SC: We may need to revisit the data modeling.
+
+ACTION: Find consensus on the solution to this, producing a proposal and make a PR by next meeting (2022-08-25)
+
+### Solid Notifications Protocol: Review notes
+URL: https://github.com/solid/notifications/issues/76
+
+* SC: Talked with Tim about it (some specifics) but I didn't process the notes/review yet.
+* SC: Can we all commit to addressing this review towards the next release of the Notifications Protocol (and WebSocket Subscription Type)?
+* SC: Tim would like a simpler protocol. The thing that Tim seems to want does not seem realistic for the next release (e.g. GET on resource and the response header would contain specific WS-related connection information)
+
+ACTION: SC and AC will process and respond to this issue (some items may already have been addressed)
+
+
+### An Alternative (WebSockets) Design
+URL: https://github.com/solid/notifications/issues/97
+
+* RG: RFC. How are we dealing with multiple-resource connections?
+* AC: Lots of ways to address that. Part of the subscription negotiation you can assert a request to listen to arbitrary number of resources. If you then as part of the subscription negotiation, say I want to subscribe to 500 or whatever resources, you can negotiate an access token, to have at least read access to those.. there are different ways to check for that too - different protocols in place. You can even write an implementation that uses id tokens from Solid OIDC. Couple your authz system and websocket system. You can absolutely have 1 connection and listen to multiple resources at the same time.
+* SC: Remind me.. Is that by way of having an array for `topic`?
+* AC: Indeed. Starting point. And then. Just need a way of negotiation is supported by the authorization server or the websocket server itself.
+* SC: ...
+* RG: One or two alternatives are possible. ??? or doing it in band in the connection itself. suppose a resource gets added, do I then have to create a separate connection or is it in band? How will it be managed for addition and removal of resources. At least we have one method. That is clearly stated.
+* AC: The way I'd do this is: first, if websockets makes use of bidirectional flow, or redo this negotiation inside the connection, then websockets is unlike any subscription mechanism. that's what's problematic about this. if however you do this on the subscription level and you want to upgrade, then you can reconnect. then whatever changed between disconnecting and connecting comes through. but for in-band it describes a separate protocol. so we should avoid that.
+* RG: Only websockets is bidirectional.
+* AC: Just because websockets has a mechanism it doesn't mean it is supported in ??? the notifications protocol is trying to be generic. in order to be generic we can't specify / customise for each.
```

Comment:
```suggestion
* AC: Just because websockets has a mechanism doesn't mean it is supported in ???. The notifications protocol is trying to be generic. To be generic, we can't specify / customise for each.
```